frontend/public/models/index.ts

```diff
@@ -254,6 +254,19 @@ export const DeploymentModel: K8sKind = {
   labelPlural: 'Deployments',
 };
 
+export const StorageClusterModel: K8sKind = {
```

Comment:
This widget should be general and work for any OLM operator with this descriptor rather than this one specific type. We shouldn't need to add this model.